api/src/test/java/com/exelaration/abstractmemery/services/ImageServiceTest.java

```diff
@@ -0,0 +1,62 @@
+package com.exelaration.abstractmemery.services;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.verify;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.Charset;
+
+import com.exelaration.abstractmemery.domains.Image;
+import com.exelaration.abstractmemery.services.implementations.ImageServiceImpl;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
+public class ImageServiceTest {
+
+    @Mock
+    private FileStorageService fileStorageService;
+
+    @Mock
+    private MetadataService metadataService;
+
+    @InjectMocks
+    private ImageServiceImpl imageService;
+
+    @BeforeEach
+    public void init() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void testSaveImageSuccessfully() throws IOException {
```

Comment:
Test names should be specific enough to convey what's being tested and what you expect.  A good rule of thumb is to use the format `FunctionUnderTest_When<SetupConditions>_Expect<Result>`